Form/Extension/DisableCSRFExtension.php

```diff
@@ -22,22 +22,18 @@
  */
 class DisableCSRFExtension extends AbstractTypeExtension
 {
-    private $securityContext;
+    private $authorizationChecker;
     private $role;
 
-    public function __construct(SecurityContextInterface $securityContext, $role)
+    public function __construct(AuthorizationChecker $authorizationChecker, $role)
```

Comment:
please never typehint the implementation but the interface (otherwise, you would not be compatible with any bundle extending the service through composition).

And you should accept both the old and the new APIs to be compatible with 2.3+ (this means removing the typehint and replacing it with 2 instanceof check in the constructor)